app/controllers/admin/application_controller.rb

```diff
@@ -41,7 +41,7 @@ class ApplicationController < ApplicationController
     private
 
     def authorization_resource
-      self.class.name.demodulize.sub("Controller", "").singularize.constantize
+      self.class.name.sub("Admin::", "").sub("Controller", "").singularize.constantize
```

Comment:
Demodulize won't work in case the controller has one more namespace apart from `Admin::`. Are there better ideas for this?

Reply:
`demodulize works fine for nested modules too:

```ruby
Admin::Something::TestsController.name.demodulize.chomp("Controller")
#=> "Tests"
```

Reply:
I meant that in this case I wanted to keep the namespace as a part of the resource name. In this case, `Users::GdprDeleteRequest`model and `Admin::Users::GdprDeleteRequestsController`.

Reply:
Oh, now I get it, that makes sense. 👍